config/webpack.config.js

```diff
@@ -195,19 +199,18 @@ const configGenerator = (buildOptions, apps) => {
   };
 
   if (isOptimizedBuild) {
-    const bucket = BUCKETS[buildOptions.buildtype];
+    baseConfig.plugins.push(new webpack.HashedModuleIdsPlugin());
+    baseConfig.mode = 'production';
+  } else {
+    const bucket = BUCKETS[buildOptions.buildtype] || '';
 
+    baseConfig.devtool = '#eval-source-map';
```

Comment:
I moved the source maps plugin so that it's enabled only for unoptimized environments. This seems like the behavior we want because traditionally only dev environments produce source maps, but if it was correct to be enabled under optimized (minified) environments, then I think we should have this enabled for all environments instead.

I was just trying to enable CSS source maps on localhost when I saw that.

Reply:
Or, thinking about it, maybe this should be localhost-only? I don't think the SCSS files are accessible beyond localhost, since they're in an tarball but not the domain bucket.

Reply:
I think we mainly use source maps in sentry so we can understand the stack trace in Sentry.

Reply:
We do want source maps enabled in prod, for the reason Rian says (and for manual production debugging, too). The difference between optimized and non-optimized environments should just be that sourcemaps are generated as external files for optimized environments and embedded in the bundle for non-optimized environments.

Reply:
Oh I think I see. `baseConfig.devtool = '#eval-source-map';` should be generating inline source maps for unoptimized builds, but it doesn't seem to be working for CSS builds.

Looks like it does work if we just remove the options from inside the `SourceMapDevToolPlugin` constructor -
![image](https://user-images.githubusercontent.com/1915775/50694800-dc1a3a80-1008-11e9-9c76-f1c0f2ed0a36.png)

Looks the same way for JS, too.